testsuite/standalone/src/test/java/org/jboss/as/test/integration/management/cli/ModuleTestCase.java

```diff
@@ -186,6 +186,20 @@ public void addRemoveModuleMetaInf() throws Exception {
         testModuleAndSlotExist(false, slot, getModulePath());
     }
 
+    @Test
+    public void addModuleWithWhitespacesInDependencies() throws Exception {
+        // should trim the surrounding whitespaces
+        try {
+            cli.sendLine("module add --name=" + MODULE_NAME
+                    + " --resources=" + jarFile.getAbsolutePath()
+                    + " --dependencies=[ org.jboss.logging ]"
```

Comment:
@spyrkob seems that you can do what you want ;-). I would remove them, but this is a minor point. Feel free to keep them or not. 
If you keep them, you should possibly add a comment that say that they are not required in this case.